content/blog/2016/2016-05-12-sf-jam-jenkins-and-azure.adoc

```diff
@@ -0,0 +1,88 @@
+infrastructure. While Brian's and my presentations are applicable to practically
+any cloud-based Jenkins environment, Thiago Almeida and Oguz Pastirmaci from
+Microsoft were also on-hand and presented some interesting Azure-specific
+offerings like
+link:https://azure.microsoft.com/en-us/services/container-service/[Azure
+Container Service] with Jenkins.
+
+To kick-off the sessions we asked attendees a few polling questions and
+received very telling responses:
+
+* How big is your Development/IT organization?
+* What is your role?
+* By show of hands do you practice?
+* At what scale (tooling and practice)?
+
+[quote, Brian Dawson]
+____
+The responses indicated that the majority of attendees were from small to
+medium organizations where they practiced Continuous Delivery across multiple
+teams. A notable 25% or greater attendees considered themselves "fullstack" or
+participating in all of the roles of Developer, QA, and Operations. Interesting
+when paired with the high number  (~80%) of the practicing CD.  Is it that
+modern team with fully implemented CD practices tend to blur the traditional
+lines of Developer, QA and Operations? In my experience this is the case for
+small to medium teams and companies but, not as much so when you begin to scale
+CD across large organizations.
+____
+
+
+After gauging the audience, Thiago and Brian reviewed Continuous Delivery (CD) and
+implementing it at scale. Highlighting that CD is being rapidly adopted
+across teams and organizations, providing: the ability to deliver faster abd
+ship to production several times per week or per day, deliver a demonstrably
+higher quality product and keep team members happier.
+
+However, when organizations fail to properly support CD as they scale, they run
+into issues such as: developers acting as administrators at the cost of
+productivity, potential lack of security and/or exposure of IP and difficulty in
+sharing best practices across teams.
+
+Thiago then highlighted that properly scaling CD practices in the organization
+along with the infrastructure itself can alleviate these issues, and discussed
+the benefits of scaling CD to on cloud platforms to provide "CD-as-a-Service."
+
+Overall I found the "theory" discussion to be on point, continuous delivery is
+not just a technology nor a people problem. Successful organizations scale their
+processes and tooling _together_.
+
+
+While we do not have video from the meetup, Brian and I did record
+link:https://channel9.msdn.com/Series/TECHPOSITORY/Scaling-Jenkins-on-Azure[a
+session] with Thiago and Oguz for link:https://channel9.msdn.com[Channel9]
+which covers much of the same content:
+
+++++
+<center>
+  <iframe src="https://channel9.msdn.com/Series/TECHPOSITORY/Scaling-Jenkins-on-Azure/player" width="640" height="360" allowFullScreen frameBorder="0"></iframe>
+</center>
+++++
+
+
+The slides from Brian's and my presentation, respectively, are linked below:
```

Comment:
"from our respective presentations," maybe. With just the link text indicating whose is whose.